src/components/Dialog/docs/examples.ts

```diff
@@ -71,3 +71,28 @@ export const mobileExample = buildExample(`
       </Dialog>
      </MobileExample>
 `);
+
+export const mobileExampleNotFullscreen = buildExample(`
+    <MobileExample>
+      <Button upgrade onClick={this.onOpenDialogButtonClick}>Open Dialog</Button>
+      <Dialog isOpen={isDialogOpen} onClose={this.onCloseDialog} notFullscreenOnMobile>
+        <div className="content" style={{ textAlign: 'center' }}>
+            <Text typography="MobileSmallTitleFont" tagName="div">Discard draft?</Text>
```

Comment:
```suggestion
            <Text typography="largeTitle" tagName="div">Discard draft?</Text>
```

I know that you just copied it from the other example but I found a bug here. `MobileSmallTitleFont` is not a valid value for `typography` and should be `largeTitle`.

Reply:
Would be happy if you could change it in the other example as well.